src/smart-components/group/group.js

```diff
@@ -25,21 +26,26 @@ const Group = ({
   isFetching,
   onDelete,
 }) => {
-  const breadcrumbsList = () => [
-    { title: 'Groups', to: '/groups' },
-    { title: isFetching ? undefined : group.name, isActive: true },
-  ];
-
   const tabItems = [
     { eventKey: 0, title: 'Roles', name: `/groups/detail/${uuid}/roles` },
     { eventKey: 1, title: 'Members', name: `/groups/detail/${uuid}/members` },
   ];
 
   const [isDropdownOpen, setDropdownOpen] = useState(false);
   const [showDefaultGroupChangedInfo, setShowDefaultGroupChangedInfo] = useState(false);
+  const [groupExists, setGroupExists] = useState(true);
+
+  const history = useHistory();
+
+  const breadcrumbsList = () => [
+    { title: 'Groups', to: '/groups' },
+    groupExists ? { title: isFetching ? undefined : group.name, isActive: true } : { title: 'Invalid group', isActive: true },
+  ];
 
   const fetchData = (apiProps) => {
-    fetchGroup(apiProps);
+    fetchGroup(apiProps).then(({ value }) => {
```

Comment:
I think it would be better to handle this in the reducer. Same for roles and users.

Reply:
Moved the error flag to reducers, much better. Thanks :-)